bin/code_climate_reek

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env ruby
+#
+# Wrapper for the CodeClimate integration.
```

Comment:
>> Since this file exists, would it make sense to not load the codeclimate-specific code when just running reek? 

But then we would have code inside `Reek` that is basically not executable on its own. So if we go down this road, we should pull all out, otherwise this would be completely confusing and quite error prone. And then we're back again at our original discussion: Integrate it properly or use a wrapper :)
I'm not saying we can't change this in the future, but for the purpose of getting our integration up and running I'd rather postpone this discussion for now.